src/promptflow/tests/sdk_cli_test/e2etests/test_flow_run.py

```diff
@@ -150,25 +152,22 @@ def test_run_bulk_error(self, pf):
         assert "Invalid variant format: v, variant should be in format of ${TUNING_NODE.VARIANT}" in str(e.value)
 
     def test_basic_evaluation(self, azure_open_ai_connection: AzureOpenAIConnection, local_client, pf):
-        data_path = f"{DATAS_DIR}/webClassification3.jsonl"
-
         result = pf.run(
-            flow=f"{FLOWS_DIR}/web_classification",
-            data=data_path,
-            column_mapping={"url": "${data.url}"},
+            flow=f"{FLOWS_DIR}/print_env_var",
+            data=f"{DATAS_DIR}/env_var_names.jsonl",
```

Comment:
why removed `column_mapping` parameter?

Reply:
I changed the flow to improve CI speed(this flow doesn't have connections), and for this flow and data, it doesn't need column mapping.